src/auth/dto/index.ts

```diff
@@ -6,3 +6,6 @@ export * from './user-request';
 export * from './create-pat-response';
 export * from './access-token-payload';
 export * from './change-password-request';
+export * from './create-qrcode-2fa-response';
+export * from './verify-2fa-request';
+export * from './recovery-keys-response';
```

Comment:
perhaps not that important, but it would be good to keep imports in alphabetical order. I wonder if there is some plugin for vscode that will automatically rearrange the imports 🤔 Perhaps typescript-hero or something similar